### PR TITLE
Audit log WIP

### PR DESCRIPTION
**Pull request by rjkip**
- [x] Revert to `surfnet/stepup-middleware-client-bundle:dev-develop` after merging https://github.com/SURFnet/Stepup-Middleware-clientbundle/pull/23

### CODE REVIEW

**Comment by rjkip** [comment]
:eyeglasses: @DRvanR

File: src/Surfnet/StepupRa/RaBundle/Service/CommandService.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+     */
+    private $commandService;
+
+    /**
+     * @var \Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface
+     */
+    private $tokenStorage;
+
+    public function __construct(
+        \Surfnet\StepupMiddlewareClientBundle\Service\CommandService $commandService,
```
Comment (DRvanR) [inline src/Surfnet/StepupRa/RaBundle/Service/CommandService.php]: Please convert to use statement

File: src/Surfnet/StepupRa/RaBundle/Service/CommandService.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        $this->commandService = $commandService;
+        $this->tokenStorage = $tokenStorage;
+    }
+
+    public function execute(Command $command)
+    {
+        /** @var \Surfnet\StepupMiddlewareClientBundle\Identity\Dto\Identity $identity */
+        $identity = $this->tokenStorage->getToken()->getUser();
+
+        return $this->commandService->execute($command, new Metadata($identity->id, $identity->institution));
```
Comment (DRvanR) [inline src/Surfnet/StepupRa/RaBundle/Service/CommandService.php]: Nice cleanup, but what about the logging and fault handling?

Comment (rjkip) [inline src/Surfnet/StepupRa/RaBundle/Service/CommandService.php]: More than the [bee-ootiful logging](https://github.com/SURFnet/Stepup-Middleware-clientbundle/blob/0cd331b498021492272dda14b57cd5aa6990eb32/src/Surfnet/StepupMiddlewareClientBundle/Service/CommandService.php#L57) there already is? If so, what faults? When there's no token/user?

---

dfd

Comment (DRvanR) [inline src/Surfnet/StepupRa/RaBundle/Service/CommandService.php]: Logging like [this][1] and [this][2]. The logging of the library is different than the logging of the application, the application is able to provider a bit more context. 

[1]: https://github.com/SURFnet/Stepup-RA/pull/52/files#diff-d80e753f094a8e1fbcfa767eecf8fafeL163
[2]: https://github.com/SURFnet/Stepup-RA/pull/52/files#diff-d80e753f094a8e1fbcfa767eecf8fafeL171

**Comment by DRvanR** [comment]
As discussed: :+1: (when the build does not fail :tongue: )

**Pull request merged**